src/index.css

```diff
@@ -27,7 +27,7 @@ form label {
 }
 
 form .column:first-child {
-  @apply pr-4;
+  @apply sm:pr-4;
```

Comment:
Sweet 👍🏽

Reply:
Thank you very much. It's been great working with you.